src/stepDefinitions/publicLinkContext.js

```diff
@@ -38,7 +38,7 @@ Then(
   }
 )
 
-Then('the fields of the last public link share response of user {string} should include', function(
+Then('the last public link share response of user {string} should include following fields', function(
```

Comment:
```suggestion
Then('the last public link share response of user {string} should include the following fields', function(
```

Can we put "the" in here? Or is that a bigger change to that will also effect lots of other steps?